### ISSUE TITLE
Accessibility Improvement: add parameter for button label

### ISSUE THREAD

**Issue by rhache** [buildo/react-cookie-banner#69]
As far as I can tell there's no way to add custom attributes to the close cookie banner button. Would be great if html attributes such as `title` or `aria-label` (ie `<button aria-label='Close cookies statement'  class='button-close'>`) could be added for screen readers where the icon isn't sufficient.

It should also be noted that the icon inside the button should have an `aria-hidden="true"` attribute
